docs/publish_book.sh

```diff
@@ -0,0 +1,5 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+cp -r "${BOOK_DIR}" "${BUILD_WORKSPACE_DIRECTORY}/book"
```

Comment:
Oh, this would have failed the docs workflow.
```suggestion
cp -r "${BOOK_DIR}" "${BUILD_WORKSPACE_DIRECTORY}/docs/book"
```